modules/@apostrophecms/admin-bar/ui/apos/components/TheAposAdminBar.vue

```diff
@@ -0,0 +1,153 @@
+<template>
```

Comment:
99% of this file is directly copied from the original in the `apostrophe` module, see below for the changes.

Reply:
You could place this file in any module, as long as it is in the `ui/apos/components` subdirectory of that module. I chose the project-level `modules/@apostrophecms/admin-bar` folder to make it easier to find this override later.